src/middlewared/middlewared/plugins/jail.py

```diff
@@ -732,45 +745,24 @@ def fstab(self, jail, options):
                     verrors.add(
                         'options.destination',
                         'Destination is not a directory, please provide a'
-                        ' valid destination'
+                        ' valid destination.'
                     )
                 elif os.listdir(destination):
                     verrors.add(
                         'options.destination',
-                        'Destination directory should be empty'
+                        'Destination directory should be empty.'
```

Comment:
```suggestion
                        'Destination directory should be empty.'
```
Is this a suggestion or requirement? If it is a suggestion, why? Are there system problems to avoid or performance improvements to gain by choosing an empty directory?

Reply:
@Mrt134 this is considered a requirement from our end as the operation of mounting on non empty directories can cause issues with the existing data they have if the mount succeeds.

Reply:
Ok, thanks for the clarification. Updated the suggestion.